src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions-ver/SolutionFile/ImportAfter/Microsoft.NET.Sdk.Solution.targets

```diff
@@ -37,4 +37,12 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <Target Name="_CheckForSolutionLevelTargetFramework"
+          BeforeTargets="Build;Publish;Clean;Store;VSTest;_MTPBuild"
+          Condition="'$(_CommandLineDefinedTargetFramework)' == 'true'">
```

Comment:
Does this condition make sense? Users will still be able to bypass that with `-p:TargetFramework=...`? Is it intended to allow the bypass? If yes, IMO deserves a comment saying it explicitly and demonstrating why.